product_docs/docs/postgres_distributed_for_kubernetes/1/tde.mdx

```diff
@@ -1,45 +1,45 @@
 ---
-title: 'Transparent Data Encryption (TDE)'
+title: 'Transparent data encryption (TDE)'
```

Comment:
This is the name of our product, hence why I would say it is a proper name that should be capitalized. 
The same applies to all the corrections of the capitalization of the product name below.

Reply:
In this context, it's not referring to a product. It's referring to a technology.